code/game/objects/items/devices/radio/headset.dm

```diff
@@ -23,9 +23,10 @@
 /obj/item/radio/headset/Initialize()
 	. = ..()
 	internal_channels.Cut()
-	for(var/T in encryption_keys)
+	for(var/i in 1 to LAZYLEN(encryption_keys))
```

Comment:
Why does this need to be numerically indexed? Bit confused about the changes.

Reply:
Also, before it'd set it to a single instance instead of a list—setting encryption_keys just never functioned before.